### ISSUE TITLE
Community Site

### ISSUE THREAD

**Issue by tas50** [fred/fredz_cookbooks#2]
Fred,

Any chance you can push the cookbook back up to the community site with the revision?

**Comment by fred** [comment]
ok, which cookbook you want me to push back?

**Comment by fred** [comment]
community site, you mean opscode.com right?

**Comment by tas50** [comment]
The opscode community site

**Comment by tas50** [comment]
Basic_networking please

**Comment by fred** [comment]
I need to do some clean ups and testing before sending to opscode comunity site. 
thanks for the contribution.
cheers

**Comment by fred** [comment]
http://community.opscode.com/cookbooks/networking_basic/

**Comment by atomic-penguin** [comment]
Yeah, I would also appreciate it if you refreshed the networking_basic cookbook on the community.opscode.com site.  I would really like to use it as a dependency for other cookbooks.

**Comment by fred** [comment]
done. cheers

**Comment by atomic-penguin** [comment]
Many thanks!

**Comment by fred** [comment]
you welcome!
